### DISCUSSION TITLE
Can anyone rewrite erthink/libmdbx in rustlang?

### DISCUSSION THREAD

**Discussion by hiqsociety** [erthink/libmdbx#348] [Q&A] [answered]
Can anyone rewrite erthink/libmdbx in rustlang?

**Comment by erthink** [comment, marked as answer]
I am sure that this is unreasonable.
MDBX was inherited too many flaws from LMDB, so it is not worth the effort to repeat all of these flaws.
Only MithrilDB is likely to be cool enough to be rewritten in Rust.

**Comment by hiqsociety** [comment]
if libmdbx rewritten in rustlang, this will be the best of the best kv store. i dont understand why that will have flaws since rustlang is more reliable and faster than c++.

hope to see that happen someday.
for mithrildb, rustlang or golang will suffice.

this is fantastic work... do really hope to see the stability / reliability written in rust.
thx for the software
